cluster-operator/src/main/java/io/strimzi/operator/cluster/model/NodeRef.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.model;
+
+/**
+ * Record used to keep reference to a Kafka node through its pod name and node ID.
+ *
+ * @param podName   Name of the pod which represents this node
+ * @param nodeId    ID of the Kafka node
+ */
+public record NodeRef(String podName, int nodeId) {
+    @Override
+    public String toString() {
+        return podName;
```

Comment:
I would expect that the toString represents somehow the whole instance in string format while here it's returning only the podName. Any specific reasons for that?

Reply:
This was already in the `PodRef` class. I can add there something like `<podName>/<nodeId>` if you want. But essentially, the node ID is part of the pod name. We just wanna safe the parsing, but there is no reason for them to be different. So it seemed unnecessry.

Reply:
I would have your suggestion in place (using `<podName>/<nodeId>`) just for consistency about the general meaning of String representation of a instance ... but it's really up to you I can live with the current one, I was just curious.

Reply:
Ok, I updated it.